packages/fcl/package.json

```diff
@@ -40,9 +42,10 @@
     "prepublishOnly": "npm test && npm run build",
     "test": "jest",
     "test:watch": "jest --watch",
-    "build": "fcl-bundle",
+    "build": "npm run lint && fcl-bundle",
```

Comment:
Should we add this to all packages? Or could linting be included as part of fcl-bundle?